package.json

```diff
@@ -75,35 +75,39 @@
 		"packages/*"
 	],
 	"devDependencies": {
```

Comment:
My understanding is that npm devDependencies (similar to Composer's) is that these aren't considered semver breaks and don't leak to the end-user.